features/reports/reports.feature

```diff
@@ -51,6 +51,12 @@ Feature: Correctly formatted reports
       9 total warnings
       """
 
+    Examples:
+      | option         |
+      |                |
+      | --sort-by none |
+      | --sort-by n    |
```

Comment:
Actually why not get rid of the complete "none" option? What do we gain from it?